docs/process/release.md

```diff
@@ -128,6 +128,14 @@ git tag 'vXX'
 git push upstream && git push upstream --tags
 ```
 
+Double check that the packages we are about to publish are versioned correctly
+by running the following command.
+Their versions are derived from the git tag that we just created.
+
+```
+yarn admin release
```

Comment:
Should this happen before the commit or tag or push? I've had to revert and then recommit or retag a few times in some releases. It would be good to sanity check before we push the release commit.

Reply:
Good call. It should be commit & tag -> check versions -> push. I've updated the order.